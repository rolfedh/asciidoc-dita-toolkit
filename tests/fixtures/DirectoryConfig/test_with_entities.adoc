= Test Document with Entity

Welcome to {PRODUCT} version {VERSION}.

This is a test procedure for installation.
