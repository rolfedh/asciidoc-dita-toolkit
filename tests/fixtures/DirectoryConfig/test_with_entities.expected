= Test Document with Entity

Welcome to Red Hat Enterprise Linux version 9.0.

This is a test procedure for installation.
